x-pack/plugins/watcher/public/application/models/watch_status/watch_status.js

```diff
@@ -15,18 +15,10 @@ export class WatchStatus {
     this.state = get(props, 'state');
     this.comment = get(props, 'comment');
     this.isActive = get(props, 'isActive');
-    this.lastFired = getMoment(get(props, 'lastFired'));
+    this.lastExecution = getMoment(get(props, 'lastExecution'));
     this.lastChecked = getMoment(get(props, 'lastChecked'));
     this.lastMetCondition = getMoment(get(props, 'lastMetCondition'));
 
-    if (this.lastFired) {
```

Comment:
I removed these properties because it didn't look like they were being used anywhere.